src/js/hmrc/mtdFraudPrevention.js

```diff
@@ -59,6 +62,10 @@ export const getFraudPreventionHeaders = async () => {
       header: fraudPreventionHeadersEnum.DEVICE_LOCAL_IPS,
       callback: async () => encodeURI(await getDeviceLocalIPAsString()),
     },
+    {
+      header: fraudPreventionHeadersEnum.DEVICE_ID,
+      callback: getDeviceId,
```

Comment:
I guess this can just be `callback: uuid,`

Reply:
done